src/client/components/DataGrid/DataColumn.tsx

```diff
@@ -6,8 +6,13 @@ import classNames from 'classnames'
 type Props = React.PropsWithChildren<{ className?: string; head?: boolean }>
 
 const DataColumn: React.FC<Props> = (props: Props) => {
-  const { children, className, head } = props
-  return <div className={classNames('data-grid-column', className, { head })}>{children}</div>
+  const { children, className, head, ...rest } = props
+  return (
+    // eslint-disable-next-line react/jsx-props-no-spreading
```

Comment:
I think it's better to have explicit props , plus better to avoid eslint ignore as much as possible

Reply:
Yes, I agree.

In this case though, this is used as a 'dumb' component therefore added the object spreading to support eg. the tooltip on parent level